bobtemplates/workspace/+package.fullname+/+package.part_1+/+package.part_2+/tests/test_workflows.py.bob

```diff
@@ -2,34 +2,83 @@ from ftw.lawgiver.tests.base import WorkflowTest
 from {{{package.fullname}}}.testing import {{{package.part_1_upper}}}_FUNCTIONAL
 from path import Path
 import ftw.workspace
+import os
 
 
-class CopyFromWorkspaceWorkflowTest(WorkflowTest):
+class CompareWorkspaceWorkflowTest(WorkflowTest):
     layer = {{{package.part_1_upper}}}_FUNCTIONAL
 
+    specification_name = 'specification.de.txt'
+
     def setUp(self):
```

Comment:
Before, the setup did copy the spec, which was a precondition of the other tests, thus it was in setup.
Now its just a regular test / assertion and does not do things, right?

I'd therefore move that to a `test_` method and remove the `setUp` altogether.